main.go

```diff
@@ -69,14 +71,24 @@ func init() {
 	metricAlertmanagerErrors.With(prometheus.Labels{"endpoint": "silences"}).Set(0)
 }
 
+func getViewURL(sub string) string {
+	u := path.Join(config.Config.WebPrefix, sub)
+	if strings.HasSuffix(sub, "/") {
+		// if sub path had trailing slash than add it here, since path.Joing will
```

Comment:
*nitpick:* There's a couple of typos: "than" -> "then" and "path.Joing" -> "path.Join".